README.md

```diff
@@ -3,21 +3,21 @@ sbt-gitbucket-plugin
 
 [![Build Status](https://travis-ci.org/gitbucket/sbt-gitbucket-plugin.svg?branch=master)](https://travis-ci.org/gitbucket/sbt-gitbucket-plugin)
 
-This sbt plugin supports GitBucket plugin development.
+This `sbt` plugin supports GitBucket plugin development.
```

Comment:
This **sbt** means sbt as a build tool, not `sbt` command. So I don't want to quote as a code.